.github/workflows/special-tests.yml

```diff
@@ -247,3 +262,50 @@ jobs:
       include-success-outputs: ${{ inputs.include-success-outputs }}
       run-coverage: ${{ inputs.run-coverage }}
       debug-resources: ${{ inputs.debug-resources }}
+
+  test-openapi-client:
```

Comment:
```suggestion
  test-python-api-client:
```

Reply:
moved the tests to `static-checks-mypy-docs.yml`